app/indexers/hyrax/work_indexer.rb

```diff
@@ -11,6 +11,7 @@ def generate_solr_document
         solr_doc['member_of_collections_ssim']    = object.member_of_collections.map(&:first_title)
         solr_doc['member_of_collection_ids_ssim'] = object.member_of_collections.map(&:id)
         solr_doc['generic_type_sim'] = ['Work']
+        solr_doc['generic_type_si'] = 'Work'
```

Comment:
This changes AF indexing which is backward incompatible.  There are queries that look for both `'generic_type_si'` and `'generic_type_sim'`.  I'll see if I can dig up an example.

Reply:
The [SolrQueryService](https://github.com/samvera/hyrax/blob/main/app/services/hyrax/solr_query_service.rb#L97-L106) might help.

Reply:
Removed this as it isn't needed.